src/Forms/TreeDropdownField.php

```diff
@@ -649,14 +666,64 @@ public function getSchemaDataDefaults()
     {
         $data = parent::getSchemaDataDefaults();
         $data['data']['urlTree'] = $this->Link('tree');
+        $data['data']['emptyTitle'] = $this->getEmptyTitle();
+        $data['data']['showRootOption'] = $this->getShowRootOption();
+        
         return $data;
     }
-
+    
     /**
+     * Sets whether to show the Empty title as the option for root
+     *
+     * @param $emptyOption
+     * @return $this
+     */
+    public function setShowRootOption($showRoot)
+    {
+        $this->showRootOption = $showRoot;
+        
+        return $this;
+    }
+    
+    /**
+     * Get whether to show the Empty title as the option for root
+     *
+     * @return bool
+     */
+    public function getShowRootOption()
```

Comment:
I think this is a duplication of `getHasEmptyDefault`, can we consolidate the API here?